src/main/scala/cromwell/engine/workflow/ExecutionStoreKey.scala

```diff
@@ -12,7 +12,12 @@ sealed trait ExecutionStoreKey {
   val parent: Option[ExecutionStoreKey]
 }
 
-case class CallKey(scope: Call, index: Option[Int], parent: Option[ExecutionStoreKey]) extends ExecutionStoreKey
+abstract class OutputKey extends ExecutionStoreKey
```

Comment:
Apologies if I'm missing something, but why an [abstract class vs. a trait](http://stackoverflow.com/questions/1991042/what-is-the-advantage-of-using-abstract-classes-instead-of-traits)?